tests/models/esm/test_modeling_tf_esm.py

```diff
@@ -262,6 +262,24 @@ def test_resize_token_embeddings(self):
     def test_save_load_after_resize_token_embeddings(self):
         pass
 
+    def test_model_common_attributes(self):
+        config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
+
+        for model_class in self.all_model_classes:
+            model = model_class(config)
```

Comment:
The modification of the modeling code is controlled by `self.config.tie_word_embeddings`, but all the models here are using the same config. Is the toggling of tying or not tying the weights tested elsewhere?

I'm quite likely missing something just judging on the diff here though

Reply:
The reason for overriding this test is that the common test expects `model.get_output_embedding()` to return a `tf.keras.layers.Layer`, but in this case I'm using a simple shared matrix created with `add_weight`, so I had to tweak the test a little. I'm actually not testing the effect of that parameter anywhere, but maybe I should, unless it's already tested somewhere in the common tests!